Formula/erlang.rb

```diff
@@ -2,10 +2,9 @@ class Erlang < Formula
   desc "Programming language for highly scalable real-time systems"
   homepage "https://www.erlang.org/"
   # Download tarball from GitHub; it is served faster than the official tarball.
-  url "https://github.com/erlang/otp/archive/OTP-23.3.4.tar.gz"
-  sha256 "adc937319227774d53f941f25fa31990f5f89a530f6cb5511d5ea609f9f18ebe"
+  url "https://github.com/erlang/otp/releases/download/OTP-24.0/otp_src_24.0.tar.gz"
```

Comment:
If we're using a release tarball anyway (which is almost definitely the right move here) we probably want to add https://erlang.org/download/otp_src_24.0.tar.gz as a mirror (or as the main URL, not sure).

Reply:
Actually the erlang.org link is super slow for me. Probs better as a mirror.

Reply:
How is preference determined?
The official erlang.org is very slow (as mentioned in comment).

Reply:
Let's stick to GitHub. The erlang.org tarball has a different checksum for me for some reason.